_posts/2016-04-19-cruk-data-hack.md

```diff
@@ -0,0 +1,60 @@
+author: Emma Nash and Aisling Foley
+categories: [hacks]
+thumbnail: /img/blog/crukhack.png
+---
+
+
+From time to time, the team here at pebble has the opportunity to take a step back from client-facing work, and undertake a ‘hack’ - we’ll typically put aside a few days or a week to work on a project that scratches an itch, or is otherwise interesting to a team of developers and designers. If we can use this time to produce something of value, then even better.
+
+A recent opportunity was presented to us by our friends at Cancer Research UK. We offered them some time, and they offered us a problem.
+
+##The Business Problem
+
+CRUK manage a database of extremely useful information, relating to the incidence and treatment of cancer across the UK. At present, this data is mostly static, and held in a database. It’s not interactive, or externally accessible.
+
+This data is used in multiple ways:
+
+* Creation of infographics
+* Public facing statistics on the CRUK website
+* Creation of profiles for distribution amongst key influencers (politicians, healthcare professionals).
+
+The process of creating these materials is relatively labour intensive for the CRUK team, as the data is mostly ‘offline’ - i.e. requiring an export to be useful and not easily accessible to organisation or the public for interpretation.
+
+We believe that the concept of ‘open data’ is very powerful for ensuring information can be shared in a useful way. By creating open data-based tools you can increase exposure and enhance citizen participation, contribute to better public services and improve the way communities work.
+
+##The Case for Open Data
+
+The recent publication of the Open government Manifesto includes a focus on open data:
+
+>“Data has become an increasingly important resource for understanding society and government in the twenty-first century, and developing new and more responsive services for citizens.”
+
+Taking inspiration from other open data initiatives, we used the data made available by CRUK to power an API. This had the potential to address some key pain points within the organisation, and also enable other digital initiatives, namely:
+
+* Transparency and democratic control
+* Participation
+* Self-empowerment
+* Improved or new private products and services
+* Innovation
+* Improved efficiency of government services
+* Improved effectiveness of government services
+* Impact measurement of policies
+* New knowledge from combined data sources and patterns in large data volumes
+
+Specifically for Cancer Research UK, reducing the effort required to produce useful profiles and infographics is of benefit to the organisation - increasing the freshness of data available, whilst reducing the effort required to share it in a useful way. By creating an API on top of the existing data, profiles and infographics can be automatically generated and updated with the most recent information, without having the re-engage designers.
```

Comment:
@dave-hillier Does it look the same on your screen? Might have to live with it....